workspaces/repo-tools/packages/cli/src/lib/workspaces/templates/workspace/package.json.hbs

```diff
@@ -18,7 +18,8 @@
     "lint": "backstage-cli repo lint --since origin/main",
     "lint:all": "backstage-cli repo lint",
     "prettier:check": "prettier --check .",
-    "new": "backstage-cli new --scope @backstage-community"
+    "new": "backstage-cli new --scope @backstage-community",
+    "postinstall": "cd ../../ && yarn install"
```

Comment:
@benjdlambert @freben this turned out to be another weird quirk with working with our mono-repo of mono-repos setup 😅 

Since each plugin works within its own workspace, there's no reason/guarantee that someone will run `yarn install` in this repos root - they only need to run `yarn install` in the plugin workspace that they are working in. This means that `husky` might not be setup for them locally. Adding this `postinstall` to each workspace to ensure the root `yarn install` is run is a possible (though a bit awkward) workaround for this. 

Thoughts? If we think this is acceptable then I can go ahead and add this to all the other existing workspace scripts or let me know if there's a better way to do this?

Reply:
How does it work if we push this husky setup down to each of the workspaces individually rather than at the top level? Wondering if that's a better solution, but maybe it gets confused and runs the wrong hooks? Not sure.

Reply:
hm not too sure if that would work the way we want it to - I believe the `husky` setup needs to be done in the git root dir (where `.git` exists). According to https://typicode.github.io/husky/how-to.html#project-not-in-git-root-directory we could do something like `"postinstall": "cd ../../ && husky"` but I think you need to make sure `yarn install` is run at the root so that husky and the eslint/prettier deps are available from that context.

There may be a way to "hack" it so husky resolves and runs hooks against the specific workspace context without needing to run install at root but it could potentially get confused in scenarios where someone works on multiple workspaces and the hook only runs against the last installed workspace.